common/components/constants/ProjectConstants.js

```diff
@@ -10,4 +10,6 @@ export const Locations = {
     "Baltimore, MD",
     "Other"
   ]
-};
\ No newline at end of file
+};
+
+export const CoOwnerHeading = "CO-OWNERS"
```

Comment:
We might as well put this in VolunteerSection.jsx since it isn't used outside of that file

Reply:
Done.  Put it at the top after the imports. Or should it be in class scope?

Reply:
That's a good spot for it, I think.